.gitignore

```diff
@@ -5,7 +5,6 @@
 
 # FluentHub custom
 AppCredentials.config
-lib/octokit.graphql.net/
```

Comment:
@onein528 I need this. Otherwise, my Git config is ruined and using Git is impossible.

Reply:
Hey. You shouldn't do that unless you are not going to remove before merging.

Reply:
> I think it is your business. Shouldn't affect entire project. Cannot you re-clone and check out `wasdk`?

I will see.